drag_and_drop_v2/templates/html/drag_and_drop_edit.html

```diff
@@ -64,6 +64,18 @@
                 <h3>Items</h3>
             </header>
             <section class="tab-content">
+                <section class="tab-content item-background-color-form">
+                    <label>
+                        Background color for items:
+                        <input class="item-background-color" type="text" value="{{ self.item_background_color}}">
+                    </label>
+                </section>
+                <section class="tab-content item-text-color-form">
+                    <label>
+                        Text color for items:
+                        <input class="item-text-color" type="text" value="{{ self.item_text_color}}">
```

Comment:
I don't think it looks good how these inputs are not aligned:
![screen shot 2015-12-02 at 3 29 20 pm](https://cloud.githubusercontent.com/assets/945577/11547536/7b1bd8f2-9909-11e5-83e7-be7eb54be453.png)

Can you convert them to the same appearance/alignment as the fields on the first step?
![screen shot 2015-12-02 at 3 29 14 pm](https://cloud.githubusercontent.com/assets/945577/11547544/8865d7d8-9909-11e5-99f2-21acf9ea2ab7.png)

Also, it's not obvious to non-programmers what sort of values can be put into this field. Can you please add examples ("e.g. 'red' or '#ff0000'") and/or help text, either as a `placeholder` or as small text underneath the field?